data/en/mbs_0867.json

```diff
@@ -0,0 +1,629 @@
+                                    "The information you provide contributes to <a rel='noopener noreferrer' target='_blank' href='https://www.ons.gov.uk/economy/grossdomesticproductgdp'>Gross Domestic Product (GDP).</a>"
+                                ]
+                            }]
+                        }]
+                    },
+                    {
+                        "id": "reporting-period-block",
+                        "type": "Question",
+                        "questions": [{
+                            "description": "",
+                            "answers": [{
+                                "options": [{
+                                        "value": "Yes",
+                                        "label": "Yes"
+                                    },
+                                    {
+                                        "value": "No",
+                                        "label": "No"
+                                    }
+                                ],
+                                "id": "reporting-period-answer",
+                                "label": "",
+                                "description": "",
+                                "type": "Radio",
+                                "q_code": "d12",
+                                "mandatory": true
+                            }],
+                            "type": "General",
+                            "id": "reporting-period-question",
+                            "title": "Are you able to report for the period from {{exercise.start_date|format_date}} to {{exercise.end_date|format_date}}?"
+                        }],
+                        "routing_rules": [{
+                                "goto": {
+                                    "when": [{
+                                        "id": "reporting-period-answer",
+                                        "value": "Yes",
+                                        "condition": "equals"
+                                    }],
+                                    "block": "turnover-block"
+                                }
+                            },
+                            {
+                                "goto": {
+                                    "block": "reporting-period-block-2"
+                                }
+                            }
+                        ],
+                        "title": "Reporting period"
+                    },
+                    {
+                        "type": "Question",
+                        "id": "reporting-period-block-2",
+                        "title": "Reporting period",
+                        "questions": [{
+                            "type": "DateRange",
+                            "id": "reporting-period-question-2",
+                            "title": "What are the dates of the period that you will be reporting for?",
```

Comment:
Date validation?

Reply:
Waiting on date range validation to be merged.